packages/rum/src/boot/rum.entry.ts

```diff
@@ -9,5 +9,10 @@ export const datadogRum = makeRumPublicApi(startRum, recorderApi)
 
 interface BrowserWindow extends Window {
   DD_RUM?: RumPublicApi
+  DD_SYNTHETICS_INJECTED_RUM?: RumPublicApi
 }
-defineGlobal(getGlobalObject<BrowserWindow>(), 'DD_RUM', datadogRum)
+defineGlobal(
+  getGlobalObject<BrowserWindow>(),
+  buildEnv.syntheticsBundle ? 'DD_SYNTHETICS_INJECTED_RUM' : 'DD_RUM',
```

Comment:
did not we discuss about having a different name for session cookie as well?

Reply:
With the new approach where Synthetics takes precedence, it is not required anymore, since the customer instance will be ignored completely (it won't create cookies). Do you see a reason why we would need a different cookie?